definitions/infra-azuremysqlflexibleserver/summary_metrics.yml

```diff
@@ -0,0 +1,29 @@
+providerAccountName:
+  tag:
+    key: providerAccountName
+  title: Azure account
+  unit: STRING
+cpu:
+  goldenMetric: cpu
+  query:
```

Comment:
As the query field is [deprecated](https://github.com/newrelic/entity-definitions/blob/main/docs/summary_metrics.md#golden-metric-reference) - 
Would it be sufficient [to just reference the golden metrics](https://github.com/newrelic/entity-definitions/blob/main/docs/summary_metrics.md#golden-metric-reference) in this and all other summary metrics ?

Reply:
right, that would be much better. I'll update the definition.